docs/MVS_Outputs.rst

```diff
@@ -296,7 +296,7 @@ and a degree of NZE higher 1 a plus-energy system.
 As above, we apply a weighting based on Electricity Equivalent.
 
 .. math::
-        Degree of NZE &=\frac{1 + (\sum_{i} {E_{grid feedin}(i)} \cdot w_i - E_{grid consumption} (i) \cdot w_i)}{\sum_i {E_{demand, i} \cdot w_i}}
+        Degree of NZE &= 1 + \frac{(\sum_{i} {E_{grid feedin}(i)} \cdot w_i - E_{grid consumption} (i) \cdot w_i)}{\sum_i {E_{demand, i} \cdot w_i}}
```

Comment:
Can you also fix that here: https://github.com/rl-institut/multi-vector-simulator/blob/ea3194bb4600a1f2bd9adc293c917897f8f76668/src/multi_vector_simulator/E3_indicator_calculation.py#L669

Reply:
Ah yes, sure!